library/std/src/sys/unix/process/process_common.rs

```diff
@@ -253,11 +253,17 @@ impl Command {
         let maybe_env = self.env.capture_if_changed();
         maybe_env.map(|env| construct_envp(env, &mut self.saw_nul))
     }
+
     #[allow(dead_code)]
     pub fn env_saw_path(&self) -> bool {
         self.env.have_changed_path()
     }
 
+    #[allow(dead_code)]
+    pub fn program_is_path(&self) -> bool {
+        self.program.to_bytes().iter().any(|b| *b == b'/')
```

Comment:
With `contains`, it will be optimized to `memchr` internally. Plus it's shorter. :slightly_smiling_face: :golfing: 

```suggestion
        self.program.to_bytes().contains(&b'/')
```

Reply:
TIL! Will update. :)